vsts/pipelines/validation.yml

```diff
@@ -3,29 +3,41 @@ parameters:
     type: object
     default:
       - 
-        key: Latest
-        value: latest
+        imageTypeDisplay: Latest
+        imageType: latest
+        imageTypeTest: latest
       - 
-        key: LtsVersions
-        value: ltsversions
+        imageTypeDisplay: LtsVersions
+        imageType: ltsversions
+        imageTypeTest: ltsversions
       - 
-        key: Jamstack
-        value: jamstack
+        imageTypeDisplay: Jamstack
+        imageType: jamstack
+        imageTypeTest: jamstack
       - 
-        key: GithubActions
-        value: githubactions
+        imageTypeDisplay: GithubActions
+        imageType: githubactions
+        imageTypeTest: githubactions
       - 
-        key: VsoFocal
-        value: vso-focal
+        imageTypeDisplay: VsoFocal
+        imageType: vso-focal
+        imageTypeTest: vso-focal
       - 
-        key: Full
-        value: full
+        imageTypeDisplay: VsoFocal2
+        imageType: vso-focal
+        imageTypeTest: vso-focal-2
```

Comment:
Note: `imageTypeTest` is introducedsince we need `imageType` to stay consistent for the buildBuildImages.sh script. And we need `dotnet test --filter category` in testBuildImages.sh script needs to have a different value